go/vt/vtgate/filters/filters.go

```diff
@@ -0,0 +1,16 @@
+package filters
+
+import (
+	"flag"
+
+	"vitess.io/vitess/go/flagutil"
+)
+
+// WatchKeyspaces - if provided this specifies which keyspaces should be
+// visible to a vtgate. By default the vtgate will allow access to any
+// keyspace.
+var WatchKeyspaces flagutil.StringListValue
```

Comment:
It's generally common practice in the code for the global var to be named the camelCase version of the flag for easier greppability, i.e. `keyspacesToWatch` in this case.

Reply:
✅: changing the name

On variable case: I assume leading cap is fine?

If not, what is the preference for exposing the value outside the package? My default if we want to do this would be:

```go
var keyspacesToWatch = ...
func KeyspacesToWatch() []string { return keyspacesToWatch }
```